google/cloud/bigquery/magics/magics.py

```diff
@@ -442,6 +444,12 @@ def _create_dataset_if_necessary(client, dataset_id):
         "this option's value in the context bqstorage_client_options."
     ),
 )
+@magic_arguments.argument(
+    "--no_query_cache",
+    action="store_true",
```

Comment:
I'm not understanding what this "store_true" is about, but it seems consistent with the other arguments.

Reply:
That's behavior IPython/Jupyter inherits from argparse https://docs.python.org/3/library/argparse.html#action

The gist is that if this argument is present it'll set `args.no_query_cache = True` rather than requiring a boolean parameter.